controller/tests/support/docker.py

```diff
@@ -202,21 +209,12 @@ def get_digest(self, request: Request, tag: str) -> Response:
     def _check_appropriate_accept(request: Request) -> bool:
         """Make sure that an Accept header allowing multi-architecture
         manifests is present.
+
+        Our client actually sends several of these, comma-separated.  This
+        is the first.
         """
-        accept = request.headers.get("Accept")
-        allowed = (
-            "application/vnd.docker.distribution.manifest.v2+json",
-            "application/vnd.docker.distribution.manifest.list.v2+json",
-            "application/vnd.oci.image.manifest.v1+json",
-            "application/vnd.oci.image.index.v1+json",
-        )
-        if accept:
-            alternatives = accept.split(",")
-            for alt in alternatives:
-                acc = alt[:pos] if (pos := alt.find(";") > -1) else alt
-                if acc in allowed:
-                    return True
-        return False
+        accept = request.headers.get("Accept").split(",")[0].strip()
+        return accept == "application/vnd.docker.distribution.manifest.v2+json"
```

Comment:
I still think it would make more sense to check the `Accept` header against the precise string that you're expecting to send. (Moving that into a constant would probably make that easier.)

Reply:
Mmmm maybe yeah but also the string I'm sending isn't *right* so I feel weird about that (I probably should not send application/json at lower quality factor, but I want to be liberal in what I accept).

Reply:
Leaving this one.